src/main/java/edu/wpi/grip/ui/pipeline/input/CheckboxInputSocketView.java

```diff
@@ -27,10 +29,14 @@ public CheckboxInputSocketView(EventBus eventBus, InputSocket<Boolean> socket) {
         this.getIdentifier().setContentDisplay(ContentDisplay.RIGHT);
     }
 
+    private void assignSocketValue(final Optional<Boolean> value) {
+        this.checkBox.setSelected(value.isPresent() && value.get());
```

Comment:
This method does not assign the socket value, so it shouldn't be named like this.  Better yet, this line could just not be abstracted into a separate function, since it's one line and it's only used twice